skype/skype-ps/skype/Set-CsTenantMigrationConfiguration.md

```diff
@@ -13,45 +13,33 @@ ms.reviewer:
 # Set-CsTenantMigrationConfiguration
 
 ## SYNOPSIS
-Provide the topic introduction here.
+Used to enable or disable Meeting Migration Service (MMS).
 
 ## SYNTAX
 
-### Identity (Default)
 ```
-Set-CsTenantMigrationConfiguration [-Tenant <Guid>] [-MeetingMigrationEnabled <Boolean>]
- [-ACPMeetingMigrationTriggerEnabled <Boolean>]
- [-MeetingMigrationSourceMeetingTypes <MeetingMigrationSourceMeetingTypes>]
- [-MeetingMigrationTargetMeetingTypes <MeetingMigrationTargetMeetingTypes>] [[-Identity] <XdsIdentity>]
- [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
-```
-
-### Instance
-```
-Set-CsTenantMigrationConfiguration [-Tenant <Guid>] [-MeetingMigrationEnabled <Boolean>]
- [-ACPMeetingMigrationTriggerEnabled <Boolean>]
- [-MeetingMigrationSourceMeetingTypes <MeetingMigrationSourceMeetingTypes>]
- [-MeetingMigrationTargetMeetingTypes <MeetingMigrationTargetMeetingTypes>] [-Instance <PSObject>] [-Force]
- [-WhatIf] [-Confirm] [<CommonParameters>]
+Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Object>] [-Confirm]
+ [[-Identity] <Object>] [-Tenant <Object>] [-Force] [-Instance <Object>] [-AsJob]
 ```
```

Comment:
```suggestion
### Identity (Default)
```
Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Boolean>] [-Confirm]
 [[-Identity] <XdsIdentity>] [-Tenant <Guid>] [-Force] [-AsJob]
```

### Instance
```
Set-CsTenantMigrationConfiguration [-WhatIf] [-MeetingMigrationEnabled <Boolean>] [-Confirm]
[[-Instance] <PSObject>] [-Tenant <Guid>] [-Force] [-AsJob]
```
```

Reply:
I think that Identity and Instance cannot be used together.

Reply:
well, it is the output of Platy, but I can put back the original ParamSets..